src/test/addresses_goerli.sol

```diff
@@ -416,5 +426,6 @@ contract Addresses {
         addr["DIRECT_SPARK_DAI_POOL"]           = 0x8b6Ae79852bcae012CBc2244e4ef85c61BAeCE35;
         addr["DIRECT_SPARK_DAI_PLAN"]           = 0x1fB2cF94D896bB50A17dD1Abd901172F088dF309;
         addr["DIRECT_SPARK_DAI_ORACLE"]         = 0xa07C4eDB18E4B3cfB9B94D9CD348BbF6d5a7f4c2;
+        addr["EXEC_PROXY_SPARK"]                = 0x4e847915D8a9f2Ab0cDf2FC2FD0A30428F25665d;
```

Comment:
I actually like the current name since it also tells what the thing is supposed to do (`exec` must be called on it).

So my point is - if we expect the subporxies to have the similar signature (all of them must be alike and have `exec`) then this name is good. Otherwise i would roll with `SUBPROXY_SPARK` to indicate its nature.

Do you know already if it's safe enough to expect the `exec` pattern in the future?

Reply:
Please let us know what you decide on for the naming convention as we've referred to it `EXEC_PROXY_SPARK` in the copy so this will need to be changed prior to hashing to avoid confusing voters who are checking against the code

Reply:
We're using it to `exec` right now, but I believe in the future each subDAO will have their own governance stack. At that point, we will not be required to call `exec()` from the core spells.

For that reason, I believe we should choose the most "future-proof" name possible.
For instance, we can have a convention that anything in the core chainlog starting with "SUB" is a subDAO contract. That would make it easier for consumers.

I'd say `SUBPROXY_SPARK` is a better name, but that's very subjective.

Reply:
Ok , will adjust then.

Reply:
`SUBPROXY_SPARK` is a good name, I vote for that

Reply:
I have updated the copy to `SUBPROXY_SPARK`

Reply:
Instances now use `SUBPROXY_SPARK` per exec doc. Resolving